src/ChluIPFS.js

```diff
@@ -167,6 +167,15 @@ class ChluIPFS {
         }
     }
 
+    async readReviewRecord(multihash) {
+        const dagNode = await this.ipfs.object.get(this.utils.multihashToBuffer(multihash));
```

Comment:
@fazo96 we should handle error returns here.

Reply:
Should we also look at when we are stroring the review record, i.e. the 'put' call below? I think we should catch any errors there too.

But I might be missing where the errors are being caught, so in that case, just point me to that :)

Reply:
We are not catching errors so the exception will be passed to whoever called readReviewRecord. What should we do in this case when we catch an error? At the moment the caller is handed an exception and the read fails (there should be no other consequences)